### DISCUSSION TITLE
How to import a VRML97 file in R2022a

### DISCUSSION THREAD

**Discussion by swjqq** [cyberbotics/webots#4268] [Q&A]
![Snipaste_2022-02-22_13-36-25](https://user-images.githubusercontent.com/59327429/155070145-676a0389-83b5-467c-a661-222e2f795719.png)

**Comment by omichel** [comment]
The VRML97 import feature is not available any more in Webots. You should convert your VRML97 models to .obj format to import them into Webots.

**Reply by swjqq** [reply]
Thanks! I successfully imported the model by opening the .wrl file and copying everything to the .wbt file.

**Reply by huiduan-25** [reply]
Hi Omichel, I am facing the same issue. I am not sure which kind of file I should generate from SolidWorks and import into Webots.

**Reply by omichel** [reply]
I would recommend to generate a `.dae`, `.obj` (with possibly `.mtl` files) or `.stl` file and refer to it in Webots from the [Mesh](https://cyberbotics.com/doc/reference/mesh) node or from the upcoming [CadShape](https://cyberbotics.com/doc/reference/cadshape?version=master) node (to be released in Webots R2022b).
